### PR TITLE
[@smogon/calc] Fix npm tests

### PR DESCRIPTION
**Pull request by shrianshChari**
Updated @pkmn/dex so that tests work.
Basically test now supports SV DLC 1

### CODE REVIEW

File: calc/package-lock.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
   "name": "@smogon/calc",
-  "version": "0.7.0",
+  "version": "0.8.0",
```
Comment (shrianshChari) [inline calc/package-lock.json:3]: I don't know why the version in the package-lock.json changes, I just edited the `package.json`, then did `rm -rf node_modules` and `npm i` and the version modified automatically.

Comment (monsanto) [reply calc/package-lock.json:3]: Maybe someone forgot to commit a package-lock change? This happened to me too

**Comment by shrianshChari** [comment]
Maybe. I'll try and revert the change on my end and run the tests again.

**Comment by shrianshChari** [comment]
Ok, it seems that my `node_modules` were out of date and the pre-DLC version of @pkmn/dex was installed.

**Pull request closed without merging**